src/web/src/components/Posts/GitHubMobile.tsx

```diff
@@ -0,0 +1,156 @@
+import {
+  createStyles,
+  makeStyles,
+  Theme,
+  ListSubheader,
+  Accordion,
+  Typography,
+  AccordionSummary,
+  AccordionDetails,
+} from '@material-ui/core';
+import Repos from './Repos';
+import Issues from './Issues';
+import PullRequests from './PullRequests';
+import Commits from './Commits';
+import Users from './Users';
+import { VscGithub } from 'react-icons/vsc';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      padding: '0',
+      display: 'flex',
+      borderLeft: '1.5px solid #999999',
+      flexDirection: 'column',
+      [theme.breakpoints.up('lg')]: {
+        width: '21rem',
+      },
+      color: theme.palette.text.secondary,
+    },
+    GitHubInfoContainer: {
+      margin: '2rem 0 0 1rem',
+    },
+    icon: {
+      fontSize: '4rem',
+      marginRight: '1rem',
+      verticalAlign: 'text-bottom',
+      paddingLeft: '6rem',
+      alignItems: 'right',
+    },
+    accordionSummary: {
+      width: '10 rem',
+      paddingRight: '0',
+      justifyContent: 'flex-end',
+    },
+    accordion: {
+      backgroundColor: 'rgba(255, 255, 255, 0)',
+      border: 'none',
+      boxShadow: 'none',
+      justifyContent: 'flex-end',
+    },
+    '&:before': {
+      display: 'none',
+    },
+  })
+);
+
+type Props = {
+  ghUrls: string[];
+};
+
+const filterGitHubUrls = (urls: string[]) => {
+  const issues: Set<string> = new Set();
```

Comment:
We should extract the logic for GitHub information and reuse them in both mobile and desktop components vs copying it.
Another way is to make `GithubInfo` component render the info differently using [useMediaQuery](https://mui.com/components/use-media-query/).

Reply:
I am working on this, when I get all the required information from GitHubInfo, would use ```import GitHubInfo from './GitHubInfo';``` be enough for it or any other additional step is needed?

Reply:
This has been updated. I removed those and reused them from GitHubInfo. Thanks